demos/dermatology/tests/chartFactory.js

```diff
@@ -146,6 +146,22 @@
                 });
             }
         },
+        Contour: {
+            simple: function (callback) {
+                legacyRequire(["test/DataFactory", "src/chart/Contour"], function (DataFactory, Contour) {
+                    var randomX = d3.random.normal(200, 80),
+                        randomY = d3.random.normal(200, 80),
+                        points = d3.range(2000).map(function () { return [randomX(), randomY()]; });
+
+                    callback(new Contour()
+                        .xAxisType("linear")
+                        .yAxisType("linear")
+                        .columns(DataFactory.ND.subjects.columns)
+                        .data(points)
+                    );
+                });
+            }
+        },
```

Comment:
Can you add an example of it layered as well?  (see chart/Colum/lineLayer)?

Also for the dataset you could try using the same code the HexBin uses (it might look better)?